mongo.js

```diff
@@ -24,14 +26,29 @@ Cursor.prototype._save = function ( object, callback ) {
             delete serialized.id;
         }
 
-        collection.save( serialized, ondone );
+        let isUpdate = false
+
+        if ( !serialized._id ) {
+            collection.insertOne( serialized, ondone )
+        } else {
+            isUpdate = true
+            const filter = { "_id": serialized._id }
+            const update = { $set: serialized }
+            const options = { upsert: true }
+            collection.updateOne( filter, update, options, ondone );
```

Comment:
@Offir Wait - this changes everything. It turns everything into a partial update rather then a complete one. No? Is this intentional? was it fully tested in Panoply to ensure that it doesn't break stuff? This seems risky.

Reply:
@avinoamr it was tested locally and it looks good.
Panoply doesn't use dbstream-mongo to update documents. 
It uses `collection.update`.
I only added `collection.updateOne` to keep the same functionality as `collection.save' and not to do any breaking changes.